### ISSUE TITLE
Results look bad on pretrained models

### ISSUE THREAD

**Issue by Blade6570** [zhixinshu/DeformingAutoencoders-pytorch#2]
Hi, Thanks for sharing the code. I used the pretrained model as "DAE for CelebA with Dense encoder decoder, where opt.idim = 8 (./DAE_CelebA_idim8)" and result doesn't look that promising. Is it because of some preprocessing on face image crop? or this is the expected results? Please see the link for the resulting images: https://drive.google.com/file/d/13TO13cYOcJ6cQZdnZAAaTyZ_eOrIIDf3/view?usp=sharing.

Thanks

**Comment by Blade6570** [comment]
Fixed it. It was a mistake from my side. Now the results look perfect. 

Thanks

**Comment by bhushan23** [comment]
@Blade6570 could you please elaborate what was happened and how did you fix it?
Will be helpful to log just in case someone faces similar issue in future

**Comment by Blade6570** [comment]
I was using the testing section in the script "train_DAE_CelebA.py" to test the pretrained model. The results were highly varying with the batchSize, as the model was in the training mode (BatchNorm statistics were not accurate). I just used .eval() in the model and it fixed the problem.
